extensions/crossmodel-lang/src/glsp-server/handler/add-entity-operation-handler.ts

```diff
@@ -5,19 +5,19 @@
 import { AddEntityOperation } from '@crossbreeze/protocol';
 import { Command, JsonOperationHandler, ModelState } from '@eclipse-glsp/server';
 import { injectable, inject } from 'inversify';
-import { DiagramNode, Entity } from '../../language-server/generated/ast';
-import { createNodeToEntityReference } from '../../language-server/util/ast-util';
-import { findAvailableNodeName } from '../../language-server/util/name-util';
-import { CrossModelState } from '../model/cross-model-state';
-import { CrossModelCommand } from './cross-model-command';
+import { DiagramNode, Entity } from '../../language-server/generated/ast.js';
+import { createNodeToEntityReference } from '../../language-server/util/ast-util.js';
+import { findAvailableNodeName } from '../../language-server/util/name-util.js';
+import { CrossModelState } from '../model/cross-model-state.js';
+import { CrossModelCommand } from './cross-model-command.js';
 
 /**
  * An operation handler for the 'AddEntityOperation' that resolves the referenced entity by name and places it in a new node on the diagram.
  */
 @injectable()
 export class CrossModelAddEntityOperationHandler extends JsonOperationHandler {
     override operationType = AddEntityOperation.KIND;
-    @inject(ModelState) protected override modelState: CrossModelState;
+    @inject(ModelState) protected override modelState!: CrossModelState;
```

Comment:
@martin-fleck-at Out of curiousity, what does the ! do? Is it enforcing non-nullability or something?

Reply:
Yes, exactly. I'm not sure why we but we do get a warning now that the properties are not initialized (directly or in the constructor) because they are injected. So in this case, I assert the non-nullability with `!` to avoid that warning.